src/applications/mhv-medications/tests/containers/RxBreadcrumbs.unit.spec.jsx

```diff
@@ -62,25 +53,68 @@ describe('Medications Breadcrumbs', () => {
           },
         },
         reducers,
-        initialEntries: [`/prescription/5678/documentation`],
+        initialEntries: [
+          `${medicationsUrls.subdirectories.DETAILS}/${prescriptionId}${
+            medicationsUrls.subdirectories.DOCUMENTATION
+          }`,
+        ],
       },
     );
     const breadcrumbs = screen.getByTestId('rx-breadcrumb-link');
     // get the href of the link
     const href = breadcrumbs.getAttribute('href');
-    expect(href).to.equal(`${medicationsUrls.PRESCRIPTION_DETAILS}/5678`);
+    expect(href).to.equal(
+      `${medicationsUrls.PRESCRIPTION_DETAILS}/${prescriptionId}`,
+    );
+    expect(breadcrumbs.getAttribute('text')).to.equal('Back');
   });
 
-  it('Renders breadcrumb if Rx details call fails', () => {
-    const screen = setup({
-      rx: {
-        prescriptions: {
-          prescriptionDetails: undefined,
-          apiError: true,
+  it('renders nothing on DOCUMENTATION route when display documentation content flag is OFF', () => {
+    const screen = setup(
+      {
+        featureToggles: {
+          // eslint-disable-next-line camelcase
+          mhv_medications_display_documentation_content: false,
         },
       },
+      [
+        `${medicationsUrls.subdirectories.DETAILS}/${prescriptionId}${
+          medicationsUrls.subdirectories.DOCUMENTATION
+        }`,
+      ],
+    );
+    expect(screen.queryByTestId('rx-breadcrumb-link')).to.be.null;
+    expect(screen.queryByTestId('rx-breadcrumb')).to.be.null;
+  });
+
+  it('renders breadcrumbs on BASE route', async () => {
+    const screen = setup({}, [medicationsUrls.subdirectories.BASE]);
+
+    await waitFor(() => {
+      const element = screen.container.querySelector('va-breadcrumbs');
+      expect(element).to.exist;
+      expect(element).to.have.attribute('data-testid', 'rx-breadcrumb');
+      expect(element.breadcrumbList).to.exist;
     });
-    const breadcrumbs = screen.getByTestId('rx-breadcrumb-link');
-    expect(breadcrumbs).to.exist;
+  });
+
+  it('renders breadcrumbs on REFILL route', async () => {
+    const screen = setup({}, [medicationsUrls.subdirectories.REFILL]);
+
+    await waitFor(() => {
+      const element = screen.container.querySelector('va-breadcrumbs');
+      expect(element).to.exist;
+      expect(element).to.have.attribute('data-testid', 'rx-breadcrumb');
+      expect(element.breadcrumbList).to.exist;
```

Comment:
You could also use the `screen.getByTestId` function provided by testing-library, here. No change necessary, just FYI.